test/UArchReplay.t.template

```diff
@@ -30,7 +32,6 @@ contract UArchReplayTest is Test {
 
     struct Entry {
         string path;
-        bool proof;
```

Comment:
Can you put back the `proof` field? The actual file format is like 
```
{ "path": "rv64ui-uarch-simple-steps.json", "steps": 12, "proofs": true },
```
Otherwise I can fix after you merge it.

Reply:
OK, re-added, let's try to get machine-emulator to generate some official logs as I think this right format generator is sitting on someone's laptop somewhere :)